submissions/jeffersonMormah/domManipulation/signup.js

```diff
@@ -0,0 +1,65 @@
+let signupForm = document.getElementById("form-signup");
+let firstName = document.getElementById("FirstName");
+let lastName = document.getElementById("LastName");
+let phoneNum = document.getElementById("Phone");
+let emailAddress = document.getElementById("email");
+let passWord = document.getElementById("Password");
+let confPassword = document.getElementById("conf-password");
+let nameCheck = /^[A-za-z]/;
+let checkNum =  /^[\+0-9]{8,15}$/;
+let validEmail = /^+([]?+)*@+([-]?+)*({2,6})+$/;
+let pwdVal = /(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{6,}$/;
+let firstNameInput = "Please Enter Your First Name";
+let firstNameError = "It can contain only letters";
+let lastNameInput = "Please Enter Your Last Name";
+let lastNameError = "Letters Only";
```

Comment:
You can reuse error message. I think you should use the `firstNameError` and rename it to be generic as `nameError` @Jeffbuchi

Reply:
@ToluAkin, pls give me a hint on formating js files. Thanks